docker/docker-compose.yaml

```diff
@@ -23,6 +23,9 @@ services:
       # console or api domain.
       # example: http://udify.app
       APP_URL: ''
+      # Check the URL for version updates.
+      # example: https://updates.dify.ai
+      CHECK_UPDATE_URL: ''
```

Comment:
Sorry about that, you should have the url filled in by default, not deleted by default.